editor/plugins/mesh_editor_plugin.cpp

```diff
@@ -50,10 +50,8 @@ void MeshEditor::_gui_input(Ref<InputEvent> p_event) {
 
 void MeshEditor::_notification(int p_what) {
 	if (p_what == NOTIFICATION_READY) {
-		//get_scene()->connect("node_removed",this,"_node_removed");
-
 		if (first_enter) {
-			//it's in propertyeditor so. could be moved around
+			//It's in the property editor.So, could be moved around.
```

Comment:
If fixing style, let's do it right :)
```suggestion
			// It's in the property editor. So, could be moved around.
```